packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -12,6 +12,7 @@ import { IComponentAs } from '@uifabric/utilities';
 import { IComponentStyles } from '@uifabric/foundation';
 import { ICSSPixelUnitRule } from '@uifabric/merge-styles/lib/IRawStyleBase';
 import { ICSSRule } from '@uifabric/merge-styles/lib/IRawStyleBase';
+import { IExtendedPersonaProps as IExtendedPersonaProps_2 } from 'office-ui-fabric-react/lib/SelectedItemsList';

```

Comment:
Do you know what's triggering this? Do we have two `IExtendedPersonaProps` interfaces? Seems odd a public API documenter would rename a public interface.

Reply:
I've seen it happening once when a package level import is done within an already exported module to the API surface of the same package, but unsure if this is what is causing it in here.